.github/workflows/test.yaml

```
@@ -61,6 +61,16 @@ jobs:
       - uses: ./.github/actions/setup-node
       - name: Run unit test
         run: yarn test-ci
+      - name: labeled Ready for Human Review
+        if: github.event.pull_request.draft == false && ${{ toJSON(github.event.pull_request.assignees) == '[]' }}
```

actionlint (1.7.4)

if: condition "github.event.pull_request.draft == false && ${{ toJSON(github.event.pull_request.assignees) == '[]' }}" is always evaluated to true because extra characters are around ${{ }}

(if-cond)